packages/frontend/src/redux/crossStateSelectors/selectCollectedAvailableForClaimData.js

```diff
@@ -0,0 +1,81 @@
+import BN from 'bn.js';
+import { isEmpty } from 'lodash';
+import { createSelector } from 'reselect';
+
+import { selectValidatorsFarmData } from '../slices/staking';
+import {
+    selectTokensFiatValueUSD,
+    selectTokenWhiteList,
+} from '../slices/tokenFiatValues';
+import { selectAllContractMetadata } from '../slices/tokens';
+
+const collectFarmingData = (...args) => {
+    try {
+        if (args.some((el) => isEmpty(el))) return [];
+
+        const [
+            validatorsFarmData,
+            contractMetadataByContractId,
+            tokenFiatValues,
+            tokenWhitelist,
+        ] = args;
+        const filteredFarms = Object.keys(validatorsFarmData)
+            .reduce((acc, validatorId) => {
+                return [...acc, ...validatorsFarmData[validatorId].farmRewards];
+            }, [])
+            .filter((farm) => farm.active && +farm.balance > 0);
```

Comment:
```suggestion
            .filter((farm) => +farm.balance > 0);
```

@marcinbodnar Could you check this please?

Reply:
I'm not sure about the `business` part of this solution. Don't the validators must withdraw farmed tokens when become inactive?

Reply:
We should not be filtering out inactive farms or validators. Users may still have unclaimed rewards on expired farms for example.

Reply:
Done